### ISSUE TITLE
[feature request] easy import of DESCRIPTOR, Node

### ISSUE THREAD

**Issue by zurk** [bblfsh/client-python#34]
Since it is hard to get DESCRIPTOR and Node modules, can we add it to bblfsh directly?
I suggest adding to `__main__.py` something like this:
```python
import importlib

DESCRIPTOR = importlib.import_module(
        "bblfsh.gopkg.in.bblfsh.sdk.%s.uast.generated_pb2" % bblfsh.sdkversion.VERSION).DESCRIPTOR
Node = importlib.import_module(
        "bblfsh.gopkg.in.bblfsh.sdk.%s.uast.generated_pb2" % bblfsh.sdkversion.VERSION).Node
```

Then everyone can just do `from bblfsh import Node, DESCRIPTOR`.

Can make a PR if you approve.

**Comment by zurk** [comment]
We have it as ast2vec part: https://github.com/src-d/ast2vec/blob/master/ast2vec/bblfsh_roles.py but I think it is more bblfsh related things.

**Comment by juanjux** [comment]
Yes, that's ugly as Black Death, the __init__.py file should import all those.

**Comment by zurk** [comment]
@juanjux btw, can we just somehow avoid `in` in the directory path? It also solves this problem.

**Comment by juanjux** [comment]
I tried several alternatives, using symlinks, renaming the in to in_ in the protofiles... this one is the one that causes less pain and should not matter much if we import everything from \_\_init\_\_.py.

**Comment by juanjux** [comment]
Implemented in 2.6.0.
